js/phemex.js

```diff
@@ -588,8 +588,8 @@ module.exports = class phemex extends Exchange {
         const base = this.safeCurrencyCode (baseId);
         const quote = this.safeCurrencyCode (quoteId);
         const status = this.safeString (market, 'status');
-        const precisionAmount = this.parseSafeNumber (this.safeString (market, 'baseTickSize'));
-        const precisionPrice = this.parseSafeNumber (this.safeString (market, 'quoteTickSize'));
+        const precisionAmount = this.parseNumber (this.parsePrecision (this.safeString (market, 'baseTickSize')));
```

Comment:
This PR is broken since it accidentally drops the support for phemex-specific numbers formatted with a comma-delimiter for thousands (`parseSafeNumber` for numbers containing symbol `,` like `1,300,000.00`).

Reply:
right, I have never seen `parseSafeNumber` before and underlooked it. closing this PR.

Reply:
> right, I have never seen `parseSafeNumber` before and underlooked it. closing this PR.

That is because `parseSafeNumber` is exchange-specific (it is just a helper shorthand for Phemex)